spec/index.bs

```diff
@@ -1995,23 +1995,31 @@ interface GPU {
 
                 [=Content timeline=] steps:
 
+                1. Let <var data-timeline=content>contentTimeline</var> be the current [=Content timeline=].
                 1. Let |promise| be [=a new promise=].
                 1. Issue the |initialization steps| on the [=Device timeline=] of |this|.
                 1. Return |promise|.
             </div>
             <div data-timeline=device>
                 [=Device timeline=] |initialization steps|:
 
+                1. Let |adapter| be `null`.
                 1. If the user agent chooses to return an adapter, it should:
-                    1. Create a [=valid=] [=adapter=] |adapter|, chosen according to
+                    1. Set |adapter| to a [=valid=] [=adapter=], chosen according to
                         the rules in [[#adapter-selection]] and the criteria in |options|.
 
                     1. If |adapter| meets the criteria of a [=fallback adapter=] set
                         |adapter|.{{adapter/[[fallback]]}} to `true`.
 
+                1. Issue the subsequent steps on <var data-timeline=content>contentTimeline</var>.
+            </div>
+            <div data-timeline=content>
+                [=Content timeline=] steps:
+
+                1. If |adapter| is not `null`:
                     1. [=Resolve=] |promise| with a new {{GPUAdapter}} encapsulating |adapter|.
 
-                1. Otherwise, |promise| [=resolves=] with `null`.
+                1. Otherwise, [=Resolve=] |promise| with `null`.
             </div>
```

Comment:
This looks good for now but afterward I think we should consider whether requestAdapter, and possibly requestDevice) can be simplified so they don't have device timeline steps at all